### PR TITLE
LPD-25049 build the dropdown item correctly

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-25049

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPDefinitionLinkDisplayContext.java [hunk 1]
```diff
@@ -66,17 +66,25 @@ public CPDefinitionLinkDisplayContext(
 
 	public List<DropdownItem> getBulkActionDropdownItems() {
 		return ListUtil.fromArray(
-			new FDSActionDropdownItem(
+			DropdownItemBuilder.putData(
+				"action", "bulkDeleteProductRelations"
+			).putData(
+				"cmd", Constants.DELETE
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPDefinitionLinkDisplayContext.java:72]: Constants.CMD

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**